src/wbi/timing.py

```diff
@@ -38,6 +48,8 @@ def _load_timing_file(self):
         timing["dc_offset"] = 5.0  # TODO: ??
         timing["stddev"] = 1.0
 
+        timing.set_index("frame", drop=False, inplace=True)
```

Comment:
Setting the index of a DataFrame to a column that makes most sense (`frame` for both `Timing` as well as `FrameSynchronous`) allows us very fast access to the rows and being able to do a join.